common/src/main/java/com/graphicsfuzz/common/util/ImageUtil.java

```diff
@@ -122,27 +122,20 @@ public static double comparePSNR(File file1, File file2) throws FileNotFoundExce
     return opencv_core.PSNR(image1, image2);
   }
 
-  public static boolean identicalImages(File file1, File file2) {
+  public static boolean identicalImages(File file1, File file2) throws IOException {
 
-    BufferedImage img1 = null;
-    BufferedImage img2 = null;
-
-    try {
-      img1 = ImageIO.read(file1);
-      img2 = ImageIO.read(file2);
-    } catch (IOException exception) {
-      exception.printStackTrace();
-    }
+    BufferedImage img1 = ImageIO.read(file1);
+    BufferedImage img2 = ImageIO.read(file2);
 
     int height = img1.getHeight();
     int width = img1.getWidth();
     if (img2.getHeight() != height || img2.getWidth() != width) {
       return false;
     }
 
-    for (int i = 0; i < height; i++) {
-      for (int j = 0; j < width; j++) {
-        if (img1.getRGB(i, j) != img2.getRGB(i, j)) {
+    for (int y = 0; y < height; ++y) {
```

Comment:
This is essentially a unrelated fix, but I saw some warnings when looking at this related code. And the x and y were backwards.